pkg/provider/aws/client.go

```diff
@@ -75,12 +85,14 @@ type Client interface {
 	RequestServiceQuotaIncrease(*servicequotas.RequestServiceQuotaIncreaseInput) (*servicequotas.RequestServiceQuotaIncreaseOutput, error)
 
 	// Organizations
-	ListAccountsForParentPages(input *organizations.ListAccountsForParentInput, fn func(*organizations.ListAccountsForParentOutput, bool) bool) error
```

Comment:
I think we need to make sure that this stays, otherwise we break the `account mgmt list` functionality.

Reply:
that function was taken out and refactored to make it easier for testing